web-next/app/globals.css

```
@@ -0,0 +1,960 @@
+/*
+ * Tailwind v4 theme: legacy tokens → --color-* へエイリアス。
+ * これで bg-acc / text-txt2 / border-brd 等の utility クラスが
+ * 自動生成され、Phase 8 以降の Atomic UI を Tailwind で書ける。
+ *
+ * フォント系の --font-sans/--font-mono/--font-display は next/font が
+ * <html> 要素に直接注入しており、CSS 側は var(--font-sans) で参照する。
+ * Tailwind の font-sans utility を使う場合は同名の循環参照になるため、
+ * ここでは theme にマッピングせず、必要なら font-[var(--font-sans)] 形式
+ * か専用の `--font-*` エイリアスを別途追加する方針とする。
+ */
+@theme inline {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)